docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,229 @@
+# KubeStellar Hub chart usage
+
+This documents explains how to use KubeStellar Hub chart to deploy a new instance of KubeStellar
+with a choice of user-defined KubeFlex Control Planes (CPs).
+
+The information provided is specific for the following release:
+
+```shell
+export KUBESTELLAR_VERSION=0.23.0-alpha.2
+```
+
+## Pre-requisites
+
+To install the Helm chart the only requirement is [Helm](https://helm.sh/).
+However, additional executables may be required to create/manage the cluster(s) (_e.g._, Kind and kubectl),
+to join Workload Execution Clusters (WEKs) (_e.g._, clusteradm),
+and to interact with Control Planes (_e.g._, kubectl), _etc_.
+For such purpose, a full list of executable that may be required can be fund [here](./pre-reqs.md).
+
+The setup of KubStellar via the Hub chart requires the existance of at least one cluster
+to be used for the deployment of the chart.
+
+This can be:
+
+1. A local **Kind** cluster with an ingress with SSL passthrough and a mapping to host port 9443
+
+This option is particularly useful for first time users or users that would like to have a local deployment.
+
+It is important to note that, if a Kind cluster name different from `kubeflex` is used, then its Control Plane name must be also be referenced during the Helm chart installation by setting `--set "kubeflex-controller.hostContainer=<control-plane-name>"`. The Control Plane name can be obtained by using `docker ps` to find the name of the container running the Kind cluster used for deploying KubeStellar Hub chart, _e.g._ `kubeflex-control-plane`.

```

Comment:
It gets set to a container name, not a control plane name, right? See my other comment on this Value.

Reply:
ok, done

Reply:
I still see "...hostContainer=\<control-plane-name\>". If I understand correctly, you mean to say "...hostContainer=\<control-plan-container-name\>. The control-plane-container-name is the name of the container in which kind or k3d is running the relevant control plane". And the initial qualification "if a Kind cluster" needs to be clarified to "when the hosting cluster was created by kind or k3d and its Ingress domain name is left to default to localtest.me". See https://github.com/kubestellar/kubeflex/issues/250#issuecomment-2115273144 .

Reply:
ok reworded as you suggested